lib/Sema/CSApply.cpp

```diff
@@ -5364,7 +5196,7 @@ static bool hasCurriedSelf(ConstraintSystem &cs, ConcreteDeclRef callee,
 }
 
 Expr *ExprRewriter::coerceCallArguments(
-    Expr *arg, AnyFunctionType *funcType,
+    Expr *arg, AnyFunctionType *funcType, ConcreteDeclRef callee,
```

Comment:
Nit: Could you please follow the same style here and put new argument in a separate line? At least to make it look consistent...

Reply:
Sure thing! I'll also follow clang-format's suggestion and move the params up to the opening paren.